src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -1294,7 +1294,8 @@ private static T SerializeAndDeserialize<T>(T value, string baseline, DataContra
             if (!result.Equal && !skipStringCompare)
             {
                 Console.WriteLine(result.ErrorMessage);
-                throw new Exception(string.Format("Test failed for input : {0}", value));
+                Console.WriteLine(string.Format("Test failed for input : {0}", value));
+                Assert.StrictEqual(baseline, actualOutput);
```

Comment:
Thanks for the comments. I'll use Assert and get rid of the Console.Write* here.